antipodal_gripper_controller/doc/userdoc.rst

```diff
@@ -0,0 +1,14 @@
+:github_url: https://github.com/ros-controls/ros2_controllers/blob/{REPOS_FILE_BRANCH}/antipodal_gripper_controller/doc/userdoc.rst
+
+.. _antipodal_gripper_controller_userdoc:
+
+Gripper Action Controller
+--------------------------------
+
+Controller for executing a gripper command action for simple single-dof antipodal grippers.
```

Comment:
```suggestion
Controller for executing a `AntipodalGripperCommand` action for simple antipodal grippers.
```

Robotiq 2F grippers are Antipodal and not `single-dof` even though you can command them with a single value.